packages/react-core/src/components/TextArea/TextArea.tsx

```diff
@@ -10,11 +10,13 @@ export enum TextAreResizeOrientation {
   both = 'both'
 }
 
-export interface TextAreaProps extends Omit<HTMLProps<HTMLTextAreaElement>, 'onChange' | 'ref'> {
+export interface TextAreaProps extends Omit<HTMLProps<HTMLTextAreaElement>, 'disabled' | 'onChange' | 'ref'> {
```

Comment:
I'm in favor of this and I think it warrants finally having a "breaking type change" discussion I've been postponing.